src/biz/workspace/publish.rs

```diff
@@ -74,7 +74,7 @@ fn check_collab_publish_name(publish_name: &str) -> Result<(), AppError> {
 
   // Only contain alphanumeric characters and hyphens
   for c in publish_name.chars() {
-    if !c.is_alphanumeric() && c != '-' {
+    if !c.is_alphanumeric() && c != '-' && c != '_' {
```

Comment:
should we use regex to handle it?

Reply:
If we there is more complicated pattern (like email) that we want to enforce, I think it would be worth to add it in. For now, this is good enough.